benchmark/merlin_hashtable_benchmark.cc.cu

```diff
@@ -240,6 +251,7 @@ void print_title() {
        << "| HMEM(GB) "
        << "| insert "
        << "|   find "
+       << "| export "
```

Comment:
Can you update the benchmark of A100 PCIE in Readme by using the newest code?

Reply:
Well, if you insist. Where do you benchmark? Selene?

Reply:
Anywhere is OK, but use the same type of GPU.

Reply:
Anywhere is Ok, Just align the GPU type with current one.